frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -44,12 +51,55 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
   canEnablePipelines,
 }) => {
   const [errorMessage, setErrorMessage] = React.useState('');
+  const defaultStorageClass: StorageClassKind[] = [];
+  const [clusterStorageClasses, setClusterStorageClasses] = React.useState(defaultStorageClass);
```

Comment:
[useApplicationSettings.ts](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/app/useApplicationSettings.tsx#L15) in the frontend has a time base refresh with an interval of 1h for the dashboardconfig, which sits in [App.tsx](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/app/AppContext.ts) / [AppContext](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/app/AppContext.ts). Maybe that would be a better place for the logic to get the StorageClasses ... instead of the SpawnerFooter of  New Workbench?